### PR TITLE
Client can get athletes from api

### PR DESCRIPTION
**Pull request by larathordar**
#### Description
Changes proposed in this pull request:

* methods to show list of athletes and individual athletes profile added
#### What I have learned working on this feature: 
Learned to use serialisers to add attributes to athletes

### CODE REVIEW

File: app/serializers/athlete_serializer.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class AthleteSerializer < ActiveModel::Serializer
+  #include Rails.application.routes.url_helpers
```
Comment (tochman) [inline app/serializers/athlete_serializer.rb]: This looks right

File: app/serializers/athlete_serializer.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  attributes :name, :link
+
+  def link
+    "https://votingapi.herokuapp.com/#{object.id}"
```
Comment (tochman) [inline app/serializers/athlete_serializer.rb]: Even if you hardcode it, it should be 
```
 "https://votingapi.herokuapp.com/athletes/#{object.id}"
```

File: app/serializers/athlete_serializer.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+    #tried to use URL helpers
+    #api_v1_athletes_url(object, host: 'http://localhost:3000' ) = http://localhost:3000/api/v1/athletes.1
```
Comment (tochman) [inline app/serializers/athlete_serializer.rb]: `api_v1_athletes_url(object)` should do the trick. Try it and tell me what error message you are getting if it does not work.

Comment (holgertidemand) [reply app/serializers/athlete_serializer.rb]: screen shot 2018-04-04 at 09 22 47

File: app/serializers/athlete_serializer.rb (hunk 2 above)
Comment (holgertidemand) [reply app/serializers/athlete_serializer.rb]: Ofc! thank you for pinting that out

**Review by kmthorsnes — APPROVED** [review]
Looks good to me.

File: spec/rails_helper.rb [hunk 4]
```diff
@@ -1,11 +1,11 @@
+require 'coveralls'
 require 'spec_helper'
 ENV['RAILS_ENV'] ||= 'test'
 require File.expand_path('../../config/environment', __FILE__)
 abort("The Rails environment is running in production mode!") if Rails.env.production?
 require 'rspec/rails'
 require 'jsonapi/rspec'
 require 'factory_bot'
-require 'coveralls'
 Coveralls.wear!('rails')
```
Comment (davidboll) [inline spec/rails_helper.rb]: Shouldn't this be on top?

Comment (holgertidemand) [reply spec/rails_helper.rb]: hahaha

**Review by davidboll — CHANGES_REQUESTED** [review]
Please see my comment

**Pull request merged**